api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -602,6 +603,11 @@ message ResponseMapper {
   // A per mapper `body_format` to override the :ref:`body_format <envoy_v3_api_field_extensions.filters.network.http_connection_manager.v3.LocalReplyConfig.body_format>`.
   // It will be used when this mapper is matched.
   config.core.v3.SubstitutionFormatString body_format_override = 4;
```

Comment:
Nit: since you're touching this message, you could fix the typo on line 600 (`body_foramt` -> `body_format`).

github won't let me comment on that line =/.

Reply:
Fixed as part of a separate PR https://github.com/envoyproxy/envoy/pull/12139